frontend/public/components/RBAC/bindings.jsx

```diff
@@ -292,11 +292,17 @@ export const RoleBindingsPage = ({
   showTitle = true,
   mock = false,
   staticFilters = undefined,
+  fixNamespace = false,
+  roleKind = null,
```

Comment:
I would change this to take in a prop like `createPath` and calculate it outside. We'll want to lock different things based on context. For instance, from the groups page, we'll want to lock the group instead. The logic will get messy for all the possibilities if we keep it in this component.

Reply:
I just pushed this; let me know if you want other tweaks.